amm/runtime/src/main/scala/ammonite/runtime/tools/IvyThing.scala

```diff
@@ -31,17 +34,22 @@ trait IvyConstructor{
  *
  * And transliterated into Scala. I have no idea how or why it works.
  */
-case class IvyThing(resolvers: () => List[Resolver]) {
+case class IvyThing(resolvers: () => List[Resolver], verboseIvy: Boolean = true) {
 
   case class IvyResolutionException(failed: Seq[String]) extends Exception(
     "failed to resolve ivy dependencies " + failed.mkString(", ")
   )
 
   var maxLevel = 2
+  //clear any previous ivy log files
+  rm(cwd/"ivyLogs.txt")
   Message.setDefaultLogger(new AbstractMessageLogger {
     def doEndProgress(msg: String) = Console.err.println("Done")
     def doProgress() = Console.err.print(".")
-    def log(msg: String, level: Int) =  if (level <= maxLevel) Console.err.println(msg)
+    def log(msg: String, level: Int) =  if (level <= maxLevel) verboseIvy match {
+      case true => Console.err.println(msg)
```

Comment:
Can we forward this to our `Interpreter`'s `printer`, rather than just dumping string to stderr?